pkg/kubelet/pluginmanager/cache/actual_state_of_world.go

```diff
@@ -75,6 +75,18 @@ type actualStateOfWorld struct {
 
 var _ ActualStateOfWorld = &actualStateOfWorld{}
 
+// HandlerWithName holds information for handler and the Name of the plugin
+type HandlerWithName struct {
+	Handler PluginHandler
+	Name    string
```

Comment:
should be `pluginName`?

Reply:
Since there is only one name in the struct, I think it is clear the name is plugin name.